README.md

```diff
@@ -1,3 +1,147 @@
-# Opsgenie App
+# Mattermost/OpsGenie Integration
 
-A Opsgenie app for Mattermost.
+* [Feature summary](#feature-summary)
+* [Set up](#set-up)
+    * [Installation HTTP](#installation-http)
+    * [Installation Mattermost Cloud](#installation-mattermost-cloud)
+    * [Configuration](#configuration)
+* [Admin guide](#admin-guide)
+    * [Slash commands](#slash-commands)
+* [End user guide](#end-user-guide)
+    * [Get started](#get-started)
+    * [Use /opsgenie commands](#use-genie-commands)
+* [Development environment](#development-environment)
+    * [Manual installation](#manual-installation)
+    * [Install dependencies](#install-dependencies)
+    * [Run the local development environment](#run-the-local-development-environment)
+  * [Run the local development environment with Docker](#run-the-local-development-environment-with-docker)
+
+This application allows you to integrate OpsGenie to your Mattermost instance, letting you know when a new alert is created, as well as getting notified about alert updates. Also, allows the user to create new alerts, add notes to alerts, close alerts, assign alerts, etc. without moving from the Mattermost window.
+
+# Feature summary
+
+**OpsGenie to Mattermost notifications:** Link your Mattermost channels with the OpsGenie Teams you want to see, so you and your team can get notifications about the creation and updates of each alert.
+
+# Set up
+
+## Installation HTTP
+
+To install, as a Mattermost system admin user, run the command ``/apps install http OPSGENIE_API`` in any channel. The ``/genie`` command should be available after the configuration has been successfully installed.
+
+The ``OPSGENIE_API`` should be replaced with the URL where the OpsGenie API instance is running. Example: ``/apps install http https://myapp.com/manifest.json``
```

Comment:
```suggestion
The ``MANIFEST_URL`` should be replaced with the URL where the this OpsGenie app is running. Example: ``/apps install http https://myapp.com/manifest.json``
```

Reply:
Is ``http https`` in ``/apps install http https://myapp.com/manifest.json`` correct?

Reply:
Yes, `http` is the install method, and `https` is the protocol to fetch the manifest in this case